### PR TITLE
Con 102mg ada chatbot

### PR DESCRIPTION
**Pull request by MarcGEthyca**
Closes #<issue>

### Description Of Changes

_Created a new Erasure only Connector for Ada Chatbot_

### Code Changes

* [x] _Added config, dataset, fixtures and tests to this repo_

### Steps to Confirm

* [x] Add/configure the connector in UI
* [x] Verify config and icon UI looks good
* [x] Create an Access DSR with the identity value
* [ ] Verify email sent by Fides with customer data is correct

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation:
  * [x] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  https://github.com/ethyca/fidesdocs/pull/255
  * [x] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
  https://github.com/ethyca/fidesdocs/issues/254
* [X] Issue Requirements are Met
* [X] Relevant Follow-Up Issues Created
* [X] Update `CHANGELOG.md`
* [X] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### CODE REVIEW

**Comment by MarcGEthyca** [comment]
A couple of notes:
1 - At the moment I can't verify emails are sent from the Fides admin =(
2 - I tried to undo the changes I made to .fides/fides.toml I can see my actual addition is gone but spacing changes due to 'command+s' re-arranged some lines =(. I was able to fix it, right until I saved again =/

File: data/saas/config/ada_chatbot_config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  type: ada_chatbot
+  description: A sample schema representing the Ada Chatbot connector for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/ada_chatbot
+  version: 0.1.0
+
+  connector_params:
+    - name: compliance_access_token
+      sensitive: True
+      description: The Data Compliance API value. This value can be seen by going to the Settings --> Integrations within the Admin GUI (https://developers.ada.cx/reference/authentication)
+      label: Data Compliance API
```
Comment (galvana) [inline data/saas/config/ada_chatbot_config.yml:13]: Is this the correct label for the `compliance_access_token`? The description mentions the "The Data Compliance API" but that doesn't seem right to me

**Review by galvana — CHANGES_REQUESTED** [review]
This just needs a quick check on one of the connector params

File: data/saas/config/ada_chatbot_config.yml (hunk 1 above)
Comment (MarcGEthyca) [reply data/saas/config/ada_chatbot_config.yml:13]: I used that name to match what this is called in the Ada chatbot admin interface. I thought it should match what it is called in the spot they hopefully are looking for it?

This is what it looks like 
Screenshot 2023-11-08 at 7 49 02 AM

**Review by galvana — APPROVED** [review]

**Pull request merged**